doc/nrf/releases/release-notes-2.4.4.rst

```diff
@@ -0,0 +1,91 @@
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+This patch release adds changes on top of the :ref:`nRF Connect SDK v2.4.0 <ncs_release_notes_240>`, :ref:`nRF Connect SDK v2.4.1 <ncs_release_notes_241>`, :ref:`nRF Connect SDK v2.4.2 <ncs_release_notes_242>` and :ref:`nRF Connect SDK v2.4.3 <ncs_release_notes_243>`.
+The changes have an effect on Bluetooth Mesh.
+
+See :ref:`ncs_release_notes_244_changelog` for the complete list of changes.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v2.4.4**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.4.4`_.
+
+IDE and tool support
+********************
+
+`nRF Connect for Visual Studio Code extension <nRF Connect for Visual Studio Code_>`_ is the only officially supported IDE for |NCS| v2.4.4.
+SEGGER Embedded Studio Nordic Edition is no longer tested or recommended for new projects.
+
+:ref:`gs_app_tcm`, used to install the |NCS| automatically from `nRF Connect for Desktop`_, is available for Windows, Linux, and macOS.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.4.4_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.4.4_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.4.4`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_244_changelog:
+
+Changelog
+*********
+
+Protocols
+=========
+
+
+
+
+Zephyr
+======
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``4bbd91a9083a588002d4397577863e0c54ba7038``.
+
+For a complete list of |NCS| specific commits and cherry-picked commits since v2.4.0, run the following command:
+
+.. code-block:: none
+
+   git log --oneline manifest-rev ^v3.3.99-ncs1
```

Comment:
this tag won't contain latest additions, right?

Reply:
I have kept it the same as https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/releases/release-notes-2.4.3.html#zephyr

Reply:
but we've merged stuff to zephyr this time

Reply:
Could you let me know what should be updated in this section? commits etc..

Reply:
If we tag zephyr, `v3.3.99-ncs2`, right?

Reply:
updated to v3.3.99-ncs2